cmd/autoheal/healer.go

```diff
@@ -188,6 +189,19 @@ func (h *Healer) Run(stopCh <-chan struct{}) error {
 	go server.ListenAndServe()
 	glog.Info("Web server started")
 
+	// Start a change watcher over loaded config files.
+	h.config.OpenChangeObserver()
+	defer h.config.CloseChangeObserver()
```

Comment:
I still think that this should be hidden in the implementation of the config. Why do we need to do it explicitly here?